config/rollup.config.js

```diff
@@ -57,6 +58,7 @@ export default {
         ]
       }
     }),
+    terser(),
```

Comment:
Presumably this will minify for dev and prod? Next step would be to apply it to prod only I guess using env variables

Reply:
done.

started looking at providing a sourcemap, but wouldn't this mean we'd need to start exporting the sources with the package?

think this may be a bigger discussion tbh... see conversation at https://github.com/googleapis/google-cloud-node/issues/2867